javascript-data-model/order-history.js

```diff
@@ -0,0 +1,52 @@
+// eslint-disable-next-line no-var
+var orderHistory = [
+  {
+    orderNum: '114-3941689-8772232',
+    orderDate: 'August 4, 2020',
+    total: 34,
+    shipTo: 'JS Masher',
+    deliverDate: 'Aug 8, 2020',
+    item: ['Javascript for impatient programmers'],
+    note: ['Rauschmayer, Dr. Axel'],
+    returnDate: ['Sep 7, 2020'],
+    price: [31.55]
```

Comment:
Rather than having separate arrays to store data about separate individual items, you would be better off having an array that stores all the items bought, with each item having its own individual object.